core/src/main/java/bio/terra/pearl/core/dao/BaseJdbiDao.java

```diff
@@ -299,19 +340,6 @@ protected List<T> findAllByPropertyCollection(String columnName, Collection<?> c
         );
     }
 
-    protected Stream<T> streamAllByPropertyCollection(String columnName, Collection<?> columnValues) {
```

Comment:
deleting this because returning a stream wasn't worth the complexity of handling ResultSet closed issues across transaction/service boundaries.